repository/src/main/resources/db/migration/V0002__RDM-10044_publish_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE event ADD COLUMN IF NOT EXISTS publish BOOLEAN;
```

Comment:
we need to discuss this in our meeting this morning, but I think we could just use this name:
`RDM-10044_publish_column.sql`